serveradmin/serverdb/sql_generator.py

```diff
@@ -211,7 +220,7 @@ def _containment_filter_template(attribute, filt):
     return template.format(_raw_sql_escape(value))
 
 
-def _condition_sql(servertypes, attribute, template):
+def _condition_sql(attribute, template, possible_servertype_ids=None):
     if attribute.special:
         field = attribute.special.field
         if field.startswith('_'):
```

Comment:
Is this still needed? This would match for queries like `_hostname=foobar`, right? When would that even happen?

Reply:
Good catch!